docs/getting-started/advanced-setting.asciidoc

```diff
@@ -187,15 +187,17 @@ The `securitySolution:maxUnassociatedNotes` field determines the maximum number
 
 [discrete]
 [[exclude-cold-frozen-data-rule-executions]]
-== Exclude cold and frozen data from rule executions 
+== Exclude cold and frozen data from rules 
 
-To ensure rules don't search cold and frozen data when executing, specify cold and frozen {ref}/data-tiers.html[data tiers] in the `excludedDataTiersForRuleExecution` field. Multiple data tiers must be separated by commas, for example: `data_frozen`, `data_cold`. This setting is turned off by default; turning it on can improve rule performance and reduce execution time. 
+To ensure the rules in your {kib} space exclude query results from cold and frozen tiers when executing, specify cold and frozen {ref}/data-tiers.html[data tiers] in the `excludedDataTiersForRuleExecution` field. Multiple data tiers must be separated by commas, for example: `data_frozen`, `data_cold`. This setting is turned off by default; turning it on can improve rule performance and reduce execution time. 
 
-This setting does not apply to {ml} rules. 
+This setting does not apply to {ml} rules because {ml} anomalies are not stored in cold or frozen data tiers.
 
 [TIP]
 ====
 
-This setting applies to all rules in a {kib} space. To only exclude cold and frozen data from specific rules, add a <<exclude-cold-frozen-data-individual-rules,Query DSL filter>> to the rules you want affected. 
+To only exclude cold and frozen data from specific rules, add a <<exclude-cold-frozen-data-individual-rules,Query DSL filter>> to the rules you want affected. 
 
-====
\ No newline at end of file
+====
+
+IMPORTANT: Even when the `excludedDataTiersForRuleExecution` advanced setting is enabled, indicator match, event correlation, and {esql} rules may still fail if a frozen or cold shard that matches the rule's specified index pattern is unavailable during rule executions. If failures occur, we recommend modifying the rule's index patterns to only match indices containing hot tier data.
```

Comment:
I'm probably missing something but elsewhere in the PR it says "This method is not supported for {esql} and {ml} rules.", so just wanted to make sure it should really say ESQL here as well.

Reply:
You might be referring to the [DSL filter](ttps://security-docs_bk_5962.docs-preview.app.elstc.co/guide/en/security/master/exclude-cold-frozen-data-individual-rules.html) that users can add to individual rules if they only want specific rules to exclude cold/frozen data. The filter operates similarly to the `excludedDataTiersForRuleExecution` advanced setting but has a smaller scope and can't be used for ES|QL and machine learning rules.